### DISCUSSION TITLE
Getting error while generating the report on spark

### DISCUSSION THREAD

**Discussion by AmiableAnil** [Sunbird-Obsrv/Community#161] [Installation] [answered]
@sowmya-dixit @manjudr Cloudify team is getting this error when executing the report in spark. Your inputs will be helpful here.

`{"eid":"JOB_LOG","ets":1713958810926,"ver":"3.0","mid":"2D150932122DEE5C768C806C34B5ED81","actor":{"id":"","type":"System"},"context":{"channel":"in.ekstep","pdata":{"id":"AnalyticsDataPipeline","ver":"1.0","pid":"CollectionSummaryJobV2","model":null},"env":"analytics","sid":null,"did":null,"cdata":null,"rollup":null},"object":null,"edata":{"message":"Ingestion Task Id: Map(error -> Could not resolve type id 'static-azure-blobstore' as a subtype of `org.apache.druid.data.input.FirehoseFactory`: known type ids = [clipped, combining, fixedCount, http, ingestSegment, inline, local, receiver, sql, static-s3, timed] (for POJO property 'firehose')\n at [Source: (org.eclipse.jetty.server.HttpInputOverHTTP); line: 233, column: 17] (through reference chain: org.apache.druid.indexing.common.task.IndexTask[\"spec\"]->org.apache.druid.indexing.common.task.IndexTask$IndexIngestionSpec[\"ioConfig\"]->org.apache.druid.indexing.common.task.IndexTask$IndexIOConfig[\"firehose\"]))","class":"org.sunbird.core.util.CourseUtils","level":"INFO"},"@timestamp":"2024-04-24T11:40:10+00:00"}`

cc: @rhwarrier @mohitga

**Comment by sowmya-dixit** [comment]
@AmiableAnil Can you please post more details on this issue like complete ingestion spec json, is it being run on azure cloud storage?

**Reply by AmiableAnil** [reply]
@sowmya-dixit It is in OCI env.

**Reply by sowmya-dixit** [reply, marked as answer]
@AmiableAnil - Looks like ingestion spec is configured to use azure as the error says - `Could not resolve type id 'static-azure-blobstore'`. Please check the `ioConfig` part of ingestion spec once.

**Reply by AmiableAnil** [reply]
@sowmya-dixit @manjudr Can we have a call to understand ingestion spec? Please let me know your availability once so that I can schedule a call.
